### PR TITLE
PC-47 Log user on registration

### PR DESCRIPTION
**Pull request by KenAdams777**
Implemented logic that allows a user to be logged immediately on submitting a registration form.

### CODE REVIEW

**Review by MuscularSloth — COMMENTED** [review]
A lot of code are working with comments logic. Is this prev PR or you have combined several tasks?

**Comment by KenAdams777** [comment]
> A lot of code are working with comments logic. Is this prev PR or you have combined several tasks?

Yep, donna why it appears here.

**Comment by bejalane** [comment]
hm... strange that not relevant code is here...

**Comment by bejalane** [comment]
i know why... you have created your branch from "delete comment" branch. Can be?

**Comment by KenAdams777** [comment]
> i know why... you have created your branch from "delete comment" branch. Can be?

I guess so, might be.

File: src/components/Modals/Registration/Registration.js [hunk 1]
```diff
@@ -150,7 +157,7 @@ function Registration() {
 			dispatch(fetchRegistration(newUserData));
 			setIsFormValid(false);
 			dispatch(closeRegistration());
-			alert("Check your inbox and follow the link to verify your account")
+			alert("Check your inbox and follow the link to verify your account");
```
Comment (bejalane) [inline src/components/Modals/Registration/Registration.js:160]: please remove this alert, it should be another thing. there is a bug on it: https://poetrycodeway.atlassian.net/browse/PC-48

**Review by bejalane — APPROVED** [review]
Plz fix one comment and you can merge

**Pull request merged**